guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -4086,14 +4090,13 @@
 02:25:06.107 --> 02:25:08.428
 [Stephen Frost]: And then Aldous Huxley comes along and writes
 
-02:25:09.278 --> 02:25:11.599
+02:25:09.278 --> 02:25:10.384
 [Stephen Frost]: Brave New World,
-[J Jay Couey]: Which is his brother, right?
 
-02:25:11.619 --> 02:25:12.659
-[j Jay Couey]: That's Julian's brother I mean it's...
+02:25:10.409 --> 02:25:13.006
+[j Jay Couey]: Which is his brother, right? That's Julian's brother I mean it's crazy.
 
-02:25:12.699 --> 02:25:13.479
+02:25:13.031 --> 02:25:13.878
 [Stephen Frost]: Correct.

```

Comment:
Adjusted timestamps and rolled lines together based on waveform.

Reply:
Useful when more than one person is talking at the same time. Got it.

Reply:
Yeah. It's hard enough as a human to do it so I'm not surprised that all of these issues were because the AI couldn't pick out different overlapping speakers.